frontend/kadambini-ganguly-explorer/kadambini.css

```
@@ -0,0 +1,486 @@
+
+.kg-hero-content {
+    max-width: 1100px;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1.6fr 1fr;
+    gap: 40px;
+    align-items: start;
+    animation: fadeIn 1s ease-out;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)